core/Test/Tasty/Run.hs

```diff
@@ -322,6 +323,7 @@ resolveDeps tests = checkCycles $ do
           , resultDescription = ""
           , resultShortDescription = "SKIP"
           , resultTime = 0
+          , printResultDetails = Nothing
```

Comment:
Can we use `empty` or `mempty` here? And on line 160? I see no other cases where `Nothing` is used in this file.

Reply:
In this project: `empty` is never used `Nothing` a lot. Hence I chose to use `Nothing` also for consistency.

But as I removed the maybe around the printer, the decision for `Nothing` or another form is now redundant.